nextflow_schema.json

```diff
@@ -305,14 +305,14 @@
                 "custom_config_version": {
                     "type": "string",
                     "description": "Git commit id for Institutional configs.",
-                    "default": "master",
+                    "default": "main",
```

Comment:
This should remain as `master` since it's related to nf-core.

Reply:
As above [3d6e79a](https://github.com/phac-nml/staramrnf/pull/20/commits/3d6e79a22a089010389cd526b300080787384089)